src/components/list.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import "./list.css"
+import { TodoItem } from '../page.tsx'
+
+interface ListsProps {
+  value: TodoItem;
+}
+
+  const Lists : React.FC<ListsProps> = ({value}) => {
```

Comment:
以下のようにすると、ListsProps みたいな余計な型定義は不要になります！
```suggestion
  const Lists : React.FC<TodoItem> = ({completed, title}) => {
```

Reply:
ありがとうございます！

Reply:
余計な型定義を削除しました！